docker-compose.yml

```diff
@@ -0,0 +1,38 @@
+version: '3'
+
+services:
+  db:
+    image: postgres:9.5
+    restart: always
+    environment:
+      POSTGRES_PASSWORD: f00d
+      POSTGRES_USER: ofn
+    volumes:
+      - 'postgres:/var/lib/postgresql/data'
+  web:
+    tty: true
+    stdin_open: true
+    build: .
+    ports:
+      - 3000:3000
+    volumes:
+      - .:/usr/src/app
+      - gems:/bundles
+      - ./config/database.yml:/usr/src/app/config/database.yml
+      - ./config/application.example.yml:/usr/src/app/config/application.yml
```

Comment:
Don't you need to change this line now to use the old file `application.yml.example`?

Reply:
Good catch, I have rebuild the image, the container is running, hope it is good now...